src/main/java/org/thoughtslive/jenkins/plugins/jira/service/JiraService.java

```diff
@@ -187,6 +188,15 @@ public JiraService(final Site jiraSite) {
     }
   }
 
+  public ResponseData<Object> addComment(final String issueIdorKey, final String comment, final String roleVisibility) {
```

Comment:
These method signatures would change to take just object instead: 

Add Comment:
```groovy
  public ResponseData<Object> addComment(final String issueIdorKey, final Object input) {
    try {
      return parseResponse(jiraEndPoints
          .addComment(issueIdorKey, input).execute());
    } catch (Exception e) {
      return buildErrorResponse(e);
    }
  }

```

Update Comment: 
```groovy
  public ResponseData<Object> updateComment(final String issueIdorKey, final String commentId,
      final Object input) {
    try {
      return parseResponse(
          jiraEndPoints.updateComment(issueIdorKey, commentId, input).execute());
    } catch (Exception e) {
      return buildErrorResponse(e);
    }
  }
```